NitroxClient/Communication/Packets/Processors/VehicleColorChangeProcessor.cs

```diff
@@ -2,10 +2,10 @@
 using NitroxClient.Communication.Packets.Processors.Abstract;
 using NitroxClient.MonoBehaviours;
 using NitroxClient.Unity.Helper;
-using NitroxModel.Helper;
 using NitroxModel.Packets;
 using NitroxModel_Subnautica.DataStructures;
 using UnityEngine;
+using NitroxModel.Logger;
```

Comment:
This line isn't sorted into the other usings, I highly recommend [this extension](https://marketplace.visualstudio.com/items?itemName=mynkow.FormatdocumentonSave) for VS.

Reply:
is this OK now ?